homework/03LessonThree.js

```diff
@@ -79,7 +80,7 @@ var problemSevenAnswer;
 // What is the ternary operator?  How does it differ from normal conditional statements?
 //
 // Your Answer Goes Here:
-//
+// a way to shorten code when dealing with a basic on off function.
```

Comment:
more than just on/off! I think you see this now that we've chatted some in person